### ISSUE TITLE
I want browsers to throw accessibility errors

### ISSUE THREAD

**Issue by aarongustafson** [WebWeWant/webwewant.fyi#181]
Browser developer tools throw errors for all sorts of things, including CORS errors and uncaught JavaScript errors. Why not throw errors for detectable accessibility issues, like “This interactive element has no name” or “This image has no alt text.”

From an end user perspective: missing names and `alt` text are just two examples that impact the work of people with disabilities. Current workarounds don&#39;t really exist for screenreader users (for example). Buttons with no name are in most cases unusable (because they’re indistinguishable). Images with no `alt` attribute are not perceivable.

From a developer perspective, lack of this feature impacts work too: it makes it harder to discover a fix inaccessibility. Workarounds exist: for instance, inspect the browser&#39;s accessibility tree to see what an accessible name computed to.

<hr>

https://webwewant.fyi/wants/11/
